crates/rattler/src/solver/libsolv/repo.rs

```diff
@@ -107,7 +107,7 @@ impl RepoRef {
     }
 
     /// Returns a pointer to the wrapped `ffi::Repo`
-    fn as_ptr(&self) -> NonNull<ffi::Repo> {
+    pub fn as_ptr(&self) -> NonNull<ffi::Repo> {
```

Comment:
Does this still need to be `pub`? Id rather not expose the ffi if we don't need to.

Reply:
I _think_ I'll need this later on, but I just removed it from the PR because it is unrelated 😉